src/app/components/edit-group-dialog/edit-group-dialog.component.ts

```diff
@@ -1,23 +1,41 @@
-import { Component, Inject, OnInit } from '@angular/core';
-import { FormArray, FormBuilder, FormGroup, ReactiveFormsModule, Validators } from '@angular/forms';
+import { Component, Inject, OnInit, inject } from '@angular/core';
+import { FormArray, FormBuilder, FormControl, FormGroup, ReactiveFormsModule, Validators } from '@angular/forms';
 import { Group } from '../../model/groupes/groupe';
 import { MAT_DIALOG_DATA, MatDialogModule, MatDialogRef } from '@angular/material/dialog';
 import { MatFormFieldModule } from '@angular/material/form-field';
 import { MatInputModule } from '@angular/material/input';
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
+import { AsyncPipe } from '@angular/common';
 import { GroupService } from '../../service/group.service';
 import { MatIconModule } from '@angular/material/icon';
 import { CommonModule } from '@angular/common';
+import { UserService } from '../../service/User/user.service';
+import { User } from '../../model/user/user';
+import { Observable, map, of, startWith } from 'rxjs';
 
 @Component({
   selector: 'app-edit-group-dialog',
   standalone: true,
-  imports: [ MatDialogModule,MatInputModule,ReactiveFormsModule,MatFormFieldModule,MatIconModule,CommonModule],
+  imports: [MatDialogModule, 
+            MatInputModule, 
+            ReactiveFormsModule, 
+            MatFormFieldModule, 
+            MatIconModule, 
+            CommonModule, 
+            MatAutocompleteModule,
+            AsyncPipe
+          ],
   templateUrl: './edit-group-dialog.component.html',
   styleUrl: './edit-group-dialog.component.scss'
 })
 export class EditGroupDialogComponent implements OnInit{
```

Comment:
you don't need to implement onInit anymore (GPT generated code ?)

Reply:
That's not my code